avocado/utils/asset.py

```diff
@@ -129,6 +132,21 @@ def _get_relative_dir(self, parsed_url):
                                     base_url.encode(astring.ENCODING))
         return os.path.join('by_location', base_url_hash.hexdigest())
 
+    def _create_meatadata_file(self, asset_file):
+        """
+        Creates json file with metadata.
```

Comment:
Nitpick: JSON is an acronym, so usually uppercase.

Reply:
Ok, I will change that